packages/dashboard-core-plugins/src/panels/ChartPanel.test.jsx

```diff
@@ -106,12 +106,20 @@ function callErrorFunction() {
   MockChart.mock.calls[MockChart.mock.calls.length - 1][0].onError();
 }
 
-function expectLoading() {
-  expect(screen.getAllByRole('img', { hidden: true }).length).toBe(2);
+function expectLoading(container) {
```

Comment:
I thought you were going to add a `title` to the `LoadingSpinner`? This is fine too, just means you need to pass in the container as well.